python/ray/serve/config.py

```diff
@@ -698,3 +702,24 @@ class Config:
         validate_assignment = True
         extra = "forbid"
         arbitrary_types_allowed = True
+
+
+@PublicAPI(stability="beta")
+class gRPCOptions(BaseModel):
+    port: int = -1
+    grpc_servicer_functions: List[str] = []
```

Comment:
what is the default behavior if user doesn't set the anything in this attribute? (better to have a comment)

Reply:
sg, will add some comments here. But default is not to start gRPC proxy :)